pl_bolts/models/rl/common/networks.py

```diff
@@ -335,3 +336,77 @@ def forward(self, input_x: Tensor) -> Tensor:
         noisy_weights = self.sigma_weight * self.epsilon_weight.data + self.weight
 
         return F.linear(input_x, noisy_weights, bias)
+
+
+class ActorCategorical(nn.Module):
+    """
+    Policy network, for discrete action spaces, which returns a distribution
+    and an action given an observation
+    """
+
+    def __init__(self, actor_net):
+        """
+        Args:
+            input_shape: observation shape of the environment
+            n_actions: number of discrete actions available in the environment
+        """
+        super().__init__()
+
+        self.actor_net = actor_net
+
+    def forward(self, states):
+        logits = self.actor_net(states)
+        pi = Categorical(logits=logits)
+        actions = pi.sample()
+
+        return pi, actions
+
+    def get_log_prob(self, pi: Categorical, actions: torch.Tensor):
+        """
+        Takes in a distribution and actions and returns log prob of actions
+        under the distribution
+        Args:
+            pi: torch distribution
+            actions: actions taken by distribution
+        Returns:
+            log probability of the acition under pi
```

Comment:
We need lines before sections to properly render docs.
```suggestion
        under the distribution

        Args:
            pi: torch distribution
            actions: actions taken by distribution

        Returns:
            log probability of the acition under pi
```